### PR TITLE
TW-2181: Fix apple reject 10 Dec

### PR DESCRIPTION
**Pull request by nqhhdev**
## Ticket
 - #2181 

## Resolved
- Android:

https://github.com/user-attachments/assets/4937244a-315d-4303-bd0f-11d822be091e

- IOS:

https://github.com/user-attachments/assets/241ce75f-b9ce-44ea-a829-d0337bde94f6

- Delete account

https://github.com/user-attachments/assets/19aec857-7240-405f-8a0e-23e944632cf4

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] change the message of dialog

Cc @Bobpodvalnyi

File: lib/pages/settings_dashboard/settings/settings.dart [hunk 1]
```diff
@@ -257,6 +262,30 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
...
+              maxLinesMessage: 5,
+            ) ==
+            ConfirmResult.cancel) {
+          return;
+        }
+
+        final emailUri = Uri.parse(
+          'mailto:software@linagora.com?subject=${Uri.encodeComponent(generateEmailSubject)}',
+        );
+        if (await canLaunchUrl(emailUri)) {
```
Comment (hoangdat) [inline lib/pages/settings_dashboard/settings/settings.dart:285]: please test case no email was logged in

File: lib/utils/dialog/twake_dialog.dart [hunk 2]
```diff
@@ -410,7 +414,7 @@ Future<ConfirmResult> showConfirmAlertDialog({
                                         color: LinagoraSysColors.material()
                                             .onSurfaceVariant,
                                       ),
-                              maxLines: 3,
+                              maxLines: maxLinesMessage,
```
Comment (hoangdat) [inline lib/utils/dialog/twake_dialog.dart:417]: should have default value

**Comment by nqhhdev** [comment]
> * [x]  change the message of dialog
> 

Updated


https://github.com/user-attachments/assets/cc4fdad5-4850-4b0f-9881-9c7a591f760b

File: lib/utils/dialog/twake_dialog.dart (hunk 2 above)
Comment (nqhhdev) [reply lib/utils/dialog/twake_dialog.dart]: updated

File: lib/pages/settings_dashboard/settings/settings.dart (hunk 1 above)
Comment (nqhhdev) [reply lib/pages/settings_dashboard/settings/settings.dart:291]: It's test for this case
https://github.com/user-attachments/assets/60ed4877-0e09-4ced-a482-0ded9e84f8c9

**Pull request merged**